kuma/contributions/jinja2/contributions/includes/contribution-form.html

```diff
@@ -0,0 +1,92 @@
+<div class="contribution-form contribution-banner {% if is_popover %}contribution-popover collapsed{% endif %}"{% if is_popover %} aria-expanded="false" aria-hidden="false"{% endif %}>
+    {% if is_popover %}
+        <div class="actions">
+            <button id="collapse" class="hide-collapse" aria-label="{{ _('Collapse pop over window') }}">{% include 'includes/icons/arrows/chevron-down.svg' %}</button>
+            <button id="close-cta" aria-label="{{ _('Close pop over window') }}">{% include 'includes/icons/general/close.svg' %}</button>
+        </div>
+    {% endif %}
+    <div class="column-container">
+        <img src="{{ static('img/hero-dino-blank.png') }}" class="backdrop-image" alt="" role="presentation">
```

Comment:
Nit: Please explicitly close the `img` tag `<img />`

Reply:
Fixed in latest PR.